superduperdb/misc/auto_schema.py

```diff
@@ -26,6 +30,7 @@ def register_module(module_name):
 
 register_module("superduperdb.ext.numpy.encoder")
 register_module("superduperdb.ext.torch.encoder")
+register_module("superduperdb.ext.pillow.encoder")
 
 
 DEFAULT_DATATYPE = DataType(
```

Comment:
Please change this to `get_serializer('dill', ...)`.

Reply:
Done